### PR TITLE
Modified checkbox styles for custom radio buttons

### PR DESCRIPTION
**Pull request by drewj**

### CODE REVIEW

File: src/elements/fields/RadioButtonGroupField.jsx [hunk 1]
```diff
@@ -82,12 +96,7 @@ function RadioButtonGroupField({
               padding: 0,
               lineHeight: 'normal'
             }}
-            css={{
-              'input[type="radio"]': {
-                margin: '5px 10px 5px 0',
-                position: 'static'
-              }
-            }}
+            css={{ ...composeCheckboxStyle(styles, true, 'radio') }}
```
Comment (bo-dun-1) [inline src/elements/fields/RadioButtonGroupField.jsx:99]: css={composeCheckboxStyle()}?

File: src/elements/fields/CheckboxGroupField.jsx [hunk 2]
```diff
@@ -56,7 +53,7 @@ function CheckboxGroupField({
             onClick={onClick}
             style={{
               display: 'flex',
-              alignItems: 'flex-start',
```
Comment (bo-dun-1) [inline src/elements/fields/CheckboxGroupField.jsx:59]: doens't this center the text when we should be fixing to top?

File: src/elements/fields/RadioButtonGroupField.jsx [hunk 3]
```diff
@@ -1,6 +1,13 @@
-import React, { useState } from 'react';
+import React, { useMemo, useState } from 'react';
 import ReactForm from 'react-bootstrap/Form';
 import { bootstrapStyles } from '../styles';
+import { applyCheckboxStyles, composeCheckboxStyle } from './CheckboxField';
+
+const applyCheckboxGroupStyles = (element, applyStyles) => {
+  applyStyles.addTargets(['checkboxGroup']);
```
Comment (bo-dun-1) [inline src/elements/fields/RadioButtonGroupField.jsx:7]: can we call this radioGroup

**Review by bo-dun-1 — CHANGES_REQUESTED** [review]

File: src/elements/fields/RadioButtonGroupField.jsx (hunk 1 above)
Comment (drewj) [reply src/elements/fields/RadioButtonGroupField.jsx:99]: Whoops

File: src/elements/fields/RadioButtonGroupField.jsx [hunk 4]
```diff
@@ -19,6 +26,14 @@ function RadioButtonGroupField({
   const [otherSelect, setOtherSelect] = useState({});
   const otherChecked =
     (otherSelect[servar.key] || fieldVal) && fieldVal === otherVal;
+
+  const styles = useMemo(() => {
+    applyCheckboxStyles(element, applyStyles);
```
Comment (bo-dun-1) [inline src/elements/fields/RadioButtonGroupField.jsx:31]: Can we call this something more generic? A little confusing when it looks like youre appliyng checkbox styles to a radio group

**Review by bo-dun-1 — APPROVED** [review]

**Pull request merged**